app/assets/v2/js/pages/profile-tribes.js

```diff
@@ -0,0 +1,272 @@
+/* eslint-disable no-loop-func */
+
+
+const loadDynamicScript = (callback, url, id) => {
```

Comment:
this is already on tribe-edit.js also loaded on profile so you probably have it already on that page

Reply:
i don't use tribe-edit and am not actually using the load dynamic script yet, i just hard coded on to the tribes-vue, lets leave as I intend to refactor to use this.